bookstore/tests/test_publish.py

```diff
@@ -0,0 +1,5 @@
+import pytest
+
+
+def test_hello():
+    assert 1000
```

Comment:
This seem erroneous?

Reply:
Ah...this was to keep pytest from complaining about the empty file. So I threw in this test which should get removed when we have real publish tests ;-)

Reply:
Let me remove and replace with something.